src/Apps/MyCollection/Routes/MyCollectionArtwork/MyCollectionArtwork.tsx

```diff
@@ -84,46 +74,66 @@ const MyCollectionArtwork: React.FC<MyCollectionArtworkProps> = ({
         <Column span={4}>
           <Media greaterThanOrEqual="sm">
             {!!isMyCollectionPhase3Enabled && <EditArtworkButton />}
+
+            <MyCollectionArtworkSidebarFragmentContainer artwork={artwork} />
+          </Media>
+
+          <Media lessThan="sm">
+            <MyCollectionArtworkSidebarTitleInfoFragmentContainer
+              artwork={artwork}
+            />
+            {hasInsights ? (
+              <Tabs mt={2} justifyContent="space-evenly">
```

Comment:
Maybe use justifyContent="space-between" instead?

Reply:
Space between makes it look weird 👀 
![image](https://user-images.githubusercontent.com/20655703/186916673-740b8c38-3ecf-43d8-9ad5-564759397b34.png)

Reply:
adn this is how it would look like if it is `space-around`
![image](https://user-images.githubusercontent.com/20655703/186918728-db4fff76-2d45-42e6-84ab-454e915da90d.png)

Reply:
Hm, it depends whether we want to match figma or palette
If palette, then ignore my comment :)